modules/aleph/aleph.module

```diff
@@ -142,6 +142,13 @@ function aleph_settings_form() {
     '#description' => t('Logs requests to the Aleph webservice. Sensitive information such as CPR number and PIN code is stripped from the requests.'),
   );
 
+  $form['aleph']['aleph_allowed_login_branches'] = [
+    '#type' => 'textfield',
+    '#title' => t('Branches allowed to login'),
+    '#default_value' => variable_get('aleph_allowed_login_branches', ''),
+    '#description' => t('Comma separated list with the allowed branches.'),
+  ];
+
```

Comment:
The number of configuration options here is growing

I think it would be preferable to keep this configuration close to other branch configuration e.g. `aleph_main_library`. Field groups might come in handy at some point.